# Interlingua translation for apport
# Copyright (c) 2015 Rosetta Contributors and Canonical Ltd 2015
# This file is distributed under the same license as the apport package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2015.
#
msgid ""
msgstr ""
"Project-Id-Version: apport\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2025-02-20 14:20+0100\n"
"PO-Revision-Date: 2025-04-10 19:37+0000\n"
"Last-Translator: karm <melo@carmu.com>\n"
"Language-Team: Interlingua <ia@li.org>\n"
"Language: ia\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2025-06-04 11:20+0000\n"
"X-Generator: Launchpad (build c425d5702ce6a53fe8d94ca38038ac108957fda2)\n"

#: ../apport/com.ubuntu.apport.policy.in.h:1
msgid "Collect system information"
msgstr "Colliger le informationes del systema"

#: ../apport/com.ubuntu.apport.policy.in.h:2
msgid ""
"Authentication is required to collect system information for this problem "
"report"
msgstr ""
"Pro colliger le informationes pro iste reporto de problema, necessita le "
"authentication"

#: ../apport/com.ubuntu.apport.policy.in.h:3
msgid "System problem reports"
msgstr "Reportos de problema del systema"

#: ../apport/com.ubuntu.apport.policy.in.h:4
msgid "Please enter your password to access problem reports of system programs"
msgstr ""
"Per favor Inserer le contrasigno pro acceder la reporto de problemas del "
"programmas del systema"

#: ../apport/ui.py:263
msgid "This package does not seem to be installed correctly"
msgstr "Iste pacchetto non pare esser installate correctemente"

#: ../apport/ui.py:273
#, python-format
msgid ""
"This does not seem to be an official %s package. Please retry after updating "
"the indexes of available packages, if that does not work then remove related "
"third party packages and try again."
msgstr ""
"Isto non sembla esser un pacchetto %s official. Retenta post le "
"actualisation del indices de pacchettos disponibile, si illo non functiona "
"alora remove le pacchettos de tertie partes associate e retenta."

#: ../apport/ui.py:303
#, python-format
msgid ""
"You have some obsolete package versions installed. Please upgrade the "
"following packages and check if the problem still occurs:\n"
"\n"
"%s"
msgstr ""
"Tu ha installate alcun version del pacchetto obsolete. Per favor promover al "
"pacchettos sequente e  controlar si le problema occurre ancora:\n"
"\n"
"%s"

#: ../apport/ui.py:454
msgid "unknown program"
msgstr "programma incognite"

#: ../apport/ui.py:456
#, python-format
msgid "Sorry, the program \"%s\" closed unexpectedly"
msgstr "Displacente, le programma \"%s\" claudeva inexpectatemente"

#: ../apport/ui.py:458
msgid ""
"Your computer does not have enough free memory to automatically analyze the "
"problem and send a report to the developers."
msgstr ""
"Tu computer non ha bastante memoria libere pro analysar automaticamente le "
"problema e inviar un reporto al disveloppatores."

#: ../apport/ui.py:463 ../apport/ui.py:1869
#, python-format
msgid "Problem in %s"
msgstr "Problema in %s"

#: ../apport/ui.py:519 ../apport/ui.py:536 ../apport/ui.py:702
#: ../apport/ui.py:708 ../apport/ui.py:931 ../apport/ui.py:1694
#: ../apport/ui.py:1836 ../apport/ui.py:1842
msgid "Invalid problem report"
msgstr "Reporto de problema invalide"

#: ../apport/ui.py:520
msgid "You are not allowed to access this problem report."
msgstr "Tu es permittite acceder iste reporto de problema."

#: ../apport/ui.py:528
msgid "Error"
msgstr "Error"

#: ../apport/ui.py:530
msgid "There is not enough disk space available to process this report."
msgstr ""
"Il non ha bastante spatio del disco disponibile a processar iste reporto."

#: ../apport/ui.py:566
msgid "No PID specified"
msgstr "Necun PID specificate"

#: ../apport/ui.py:567
msgid "You need to specify a PID. See --help for more information."
msgstr "Tu debe specificar un PID. Vide --help pro altere informationes."

#: ../apport/ui.py:576 ../apport/ui.py:681
msgid "Invalid PID"
msgstr "PID invalide"

#: ../apport/ui.py:576
msgid "The specified process ID does not exist."
msgstr "Le ID de processo specificate non existe."

#: ../apport/ui.py:581
msgid "Not your PID"
msgstr "Non tu PID"

#: ../apport/ui.py:582
msgid "The specified process ID does not belong to you."
msgstr "Le ID de processo specificate non pertine a te."

#: ../apport/ui.py:639
msgid "No package specified"
msgstr "Nulle pacchetto specificate"

#: ../apport/ui.py:641
msgid ""
"You need to specify a package or a PID. See --help for more information."
msgstr ""
"Tu necessita specificar un pacchetto o un PID. Vider --help pro plus de "
"informationes."

#: ../apport/ui.py:668
msgid "Permission denied"
msgstr "Permission negate"

#: ../apport/ui.py:670
msgid ""
"The specified process does not belong to you. Please run this program as the "
"process owner or as root."
msgstr ""
"Le processo specificate non pertine a te. Per favor flue iste programma como "
"le proprietario del processo o como radice."

#: ../apport/ui.py:682
msgid "The specified process ID does not belong to a program."
msgstr "Le ID del processo specificate non pertine a un programma."

#: ../apport/ui.py:703
#, python-format
msgid "Symptom script %s did not determine an affected package"
msgstr "Le script del symptoma %s non determina un pacchetto affligite"

#: ../apport/ui.py:709
#, python-format
msgid "Package %s does not exist"
msgstr "Le pacchetto %s non existe"

#: ../apport/ui.py:738 ../apport/ui.py:936 ../apport/ui.py:969
#: ../apport/ui.py:976
msgid "Cannot create report"
msgstr "On non pote crear un reporto"

#: ../apport/ui.py:753 ../apport/ui.py:807 ../apport/ui.py:824
msgid "Updating problem report"
msgstr "Ajornamento del reporto del problema"

#: ../apport/ui.py:755
msgid ""
"You are not the reporter or subscriber of this problem report, or the report "
"is a duplicate or already closed.\n"
"\n"
"Please create a new report using \"apport-bug\"."
msgstr ""
"Tu non es le reporter o le subscriber de iste reporto de problema, o le "
"reporto es un duplicato o ja clause.\n"
"\n"
"Per favor crear un nove reporto per \"apport-bug\"."

#: ../apport/ui.py:767
msgid ""
"You are not the reporter of this problem report. It is much easier to mark a "
"bug as a duplicate of another than to move your comments and attachments to "
"a new bug.\n"
"\n"
"Subsequently, we recommend that you file a new bug report using \"apport-"
"bug\" and make a comment in this bug about the one you file.\n"
"\n"
"Do you really want to proceed?"
msgstr ""
"Tu non es le reporter de iste reporto de problema. Il es multo plus facile "
"marcar un defecto como un duplicato de un altere que mover tu commentos e "
"attaccamentos a un nove defecto.\n"
"\n"
"Subsequentemente, nos recommenda que tu archiva un nove reporto de defecto "
"per \"apport-bug\" e face un commento in iste defecto circa le uno que tu "
"registra.\n"
"\n"
"Desira tu realmente proceder?"

#: ../apport/ui.py:807 ../apport/ui.py:824
msgid "No additional information collected."
msgstr "Nulle information additional colligite."

#: ../apport/ui.py:880
msgid "What kind of problem do you want to report?"
msgstr "Qual genere de problema desira tu reportar?"

#: ../apport/ui.py:899
msgid "Unknown symptom"
msgstr "Symptoma incognite"

#: ../apport/ui.py:900
#, python-format
msgid "The symptom \"%s\" is not known."
msgstr "Le symptoma \"%s\" non es note."

#: ../apport/ui.py:938
msgid ""
"The window option cannot be used on Wayland.\n"
"\n"
"Please find the window's process ID and then run 'ubuntu-bug <process ID>'.\n"
"\n"
"The process ID can be found by running the System Monitor application. In "
"the Processes tab, scroll until you find the correct application. The "
"process ID is the number listed in the ID column."
msgstr ""
"Le option del fenestra non pote esser usate sur Wayland.\n"
"\n"
"Trova le ID de processo del fenestra e pois exeque 'ubuntu-bug <process "
"ID>'.\n"
"\n"
"Le ID de processo pote esser trovate exequente le application de "
"Surveliantia de systema. In le scheda Processos, rola usque tu trova le "
"application correcte. Le ID de processo es le numero in le lista in le "
"columna ID."

#: ../apport/ui.py:954
msgid ""
"After closing this message please click on an application window to report a "
"problem about it."
msgstr ""
"Post claudite iste message, cliccar per favor su un fenestra de application "
"pro reportar un problema re illo."

#: ../apport/ui.py:970 ../apport/ui.py:977
msgid "xprop failed to determine process ID of the window"
msgstr "xprop ha mancate a determinar le ID de processo del fenestra"

#: ../apport/ui.py:992
#, python-format
msgid "%(prog)s <report number>"
msgstr "%(prog)s <numero de reporto>"

#: ../apport/ui.py:993
msgid "Specify package name."
msgstr "Specificar le nomine del pacchetto."

#: ../apport/ui.py:999 ../apport/ui.py:1127
msgid "Add an extra tag to the report. Can be specified multiple times."
msgstr "Adder un tag extra al reporto. Pote esser specificate multiple vices."

#: ../apport/ui.py:1035
#, python-format
msgid ""
"%(prog)s [options] [symptom|pid|package|program path|.apport/.crash file]"
msgstr ""
"%(prog)s [optiones] [symptoma|pid|pacchetto|route de programma|"
"file .apport/.crash]"

#: ../apport/ui.py:1046
msgid ""
"Start in bug filing mode. Requires --package and an optional --pid, or just "
"a --pid. If neither is given, display a list of known symptoms. (Implied if "
"a single argument is given.)"
msgstr ""
"Initiar in modo registration de defecto. Require le --package e un optional "
"--pid, o sol un --pid. Si non es date, displicar un lista del symptomas "
"notori. (Implicite si es date un argumento singule.)"

#: ../apport/ui.py:1055
msgid "Click a window as a target for filing a problem report."
msgstr ""
"Cliccar un fenestra qual destination pro registrar un reporto del problema."

#: ../apport/ui.py:1063
msgid "Start in bug updating mode. Can take an optional --package."
msgstr ""
"Initiar in modo ajornamento del defecto. Pote prender un --package optional."

#: ../apport/ui.py:1071
msgid ""
"File a bug report about a symptom. (Implied if symptom name is given as only "
"argument.)"
msgstr ""
"Registrar un reporto de defecto circa un symptoma. (Implicite si le nomine "
"del symptoma es date qual argumento unic.)"

#: ../apport/ui.py:1080
msgid ""
"Specify package name in --file-bug mode. This is optional if a --pid is "
"specified. (Implied if package name is given as only argument.)"
msgstr ""
"Specificar le nomine del pacchetto in modo --file-bug. Isto es optional si "
"es specificate un --pid. (Implicite si le nomine del pacchetto es date qual "
"argumento unic.)"

#: ../apport/ui.py:1091
msgid ""
"Specify a running program in --file-bug mode. If this is specified, the bug "
"report will contain more information.  (Implied if pid is given as only "
"argument.)"
msgstr ""
"Specificar un programma fluente in modo --file-bug. Si isto es specificate, "
"le reporto de defecto continera plus de informationes.  (Implicite si le pid "
"es date qual argumento unic.)"

#: ../apport/ui.py:1099
msgid "The provided pid is a hanging application."
msgstr "Le pid supplite es un application suspendite."

#: ../apport/ui.py:1107
#, python-format
msgid ""
"Report the crash from given .apport or .crash file instead of the pending "
"ones in %s. (Implied if file is given as only argument.)"
msgstr ""
"Reportar le crash per le file .apport o .crash date in vice de illos "
"pendente in %s. (Implicite si le file es date qual argumento unic.)"

#: ../apport/ui.py:1117
msgid ""
"In bug filing mode, save the collected information into a file instead of "
"reporting it. This file can then be reported later on from a different "
"machine."
msgstr ""
"In modo registration de defecto, salvar le informationes colligite in un "
"file in vice de reportar lo. Iste file pote pois esser reportate ex un "
"machina differente."

#: ../apport/ui.py:1133
msgid "Print the Apport version number."
msgstr "Imprimer le numero de version de Apport."

#: ../apport/ui.py:1292
msgid ""
"This will launch apport-retrace in a terminal window to examine the crash."
msgstr ""
"Isto lanceara apport-retrace in un fenestra de terminal pro examinar le "
"crash."

#: ../apport/ui.py:1296
msgid "Run gdb session"
msgstr "Facer fluer un session gdb"

#: ../apport/ui.py:1297
msgid "Run gdb session without downloading debug symbols"
msgstr "Facer fluer un session gdb sin discargar symbolos del correction"

#. TRANSLATORS: %s contains the crash report file name
#: ../apport/ui.py:1299
#, python-format
msgid "Update %s with fully symbolic stack trace"
msgstr "Ajornar %s con le tracia del pila plenmente symbolic"

#: ../apport/ui.py:1353
msgid ""
"Saving crash reporting state failed. Can't set auto or never reporting mode."
msgstr ""
"Le reservation del stato de reporto de crash falleva. Impossibile predefinir "
"modo de reportar automatic o jammais."

#: ../apport/ui.py:1357
msgid "Can't remember send report status settings"
msgstr "Impossibile rememorar parametros del stato de invio reportos"

#: ../apport/ui.py:1438 ../apport/ui.py:1451
msgid ""
"This problem report applies to a program which is not installed any more."
msgstr ""
"Iste reporto de problema refere se a un programma que non es plus installate."

#: ../apport/ui.py:1471
#, python-format
msgid ""
"The problem happened with the program %s which changed since the crash "
"occurred."
msgstr ""
"Le problema ha occurrite con le programma %s que ha cambiate depois que le "
"crash occurreva."

#. can happen with broken core dumps
#. can happen with broken gz values
#: ../apport/ui.py:1530 ../apport/ui.py:1642 ../apport/ui.py:1840
msgid "This problem report is damaged and cannot be processed."
msgstr "Iste reporto de problema is damnificate e non pote esser processate."

#: ../apport/ui.py:1536
msgid "This report is about a package that is not installed."
msgstr "Iste reporto es re un pacchetto que non es installate"

#: ../apport/ui.py:1543
msgid "An error occurred while attempting to process this problem report:"
msgstr ""
"Un error occurreva durante que tentava processar iste reporto de problema:"

#: ../apport/ui.py:1560
msgid ""
"You have two versions of this application installed, which one do you want "
"to report a bug against?"
msgstr ""
"Tu ha duo versiones installate de iste application, contra qual vole tu "
"reportar un error?"

#: ../apport/ui.py:1565
#, python-format
msgid "%s snap"
msgstr "%s snap"

#: ../apport/ui.py:1566
#, python-format
msgid "%s deb package"
msgstr "%s pacchetto deb"

#: ../apport/ui.py:1604
#, python-format
msgid "%s is provided by a snap published by %s. Contact them via %s for help."
msgstr ""
"%s es fornite per un snap publicate per %s. Contacta illes via %s pro adjuta."

#: ../apport/ui.py:1609
#, python-format
msgid ""
"%s is provided by a snap published by %s. No contact address has been "
"provided; visit the forum at https://forum.snapcraft.io/ for help."
msgstr ""
"%s es fornite per un snap publicate per %s. Nulle adresse de contacto ha "
"essite fornite; visita le forum a https://forum.snapcraft.io/ pro le adjuta."

#: ../apport/ui.py:1695
msgid "Could not determine the package or source package name."
msgstr "Non poterea determinar le pacchetto o le nomine del pacchetto fonte."

#: ../apport/ui.py:1726
msgid "Unable to start web browser"
msgstr "Incapace de lancear le navigator del web"

#: ../apport/ui.py:1727
#, python-format
msgid "Unable to start web browser to open %s."
msgstr "Incapace de lancear le navigator del web pro aperir %s."

#: ../apport/ui.py:1798
msgid ""
"Cannot connect to crash database, please check your Internet connection."
msgstr ""
"On non pote connecter al base de datos del crash, per favor controlar le "
"connexion al Rete."

#: ../apport/ui.py:1801
msgid "Network problem"
msgstr "Problema de rete"

#: ../apport/ui.py:1827
msgid "Memory exhaustion"
msgstr "Exhaustion del memoria"

#: ../apport/ui.py:1829
msgid "Your system does not have enough memory to process this crash report."
msgstr ""
"Tu systema non ha bastante memoria pro processar iste reporto de crash."

#: ../apport/ui.py:1874
#, python-format
msgid ""
"The problem cannot be reported:\n"
"\n"
"%s"
msgstr ""
"Le problema non pote esser reportate:\n"
"\n"
"%s"

#: ../apport/ui.py:1928 ../apport/ui.py:1940
msgid "Problem already known"
msgstr "Problema ja note"

#: ../apport/ui.py:1930
msgid ""
"This problem was already reported in the bug report displayed in the web "
"browser. Please check if you can add any further information that might be "
"helpful for the developers."
msgstr ""
"Iste problema esseva ja reportate in le reporto de defecto displicate in le "
"navigator del Web. Per favor controla si tu pote adder alcun information "
"additional que pote esser auxilio al disveloppatores."

#: ../apport/ui.py:1941
msgid "This problem was already reported to developers. Thank you!"
msgstr "Iste problema esseva ja reportate al disveloppatores. Gratias!"

#: ../bin/apport-cli.py:87
msgid "Press any key to continue..."
msgstr "Pulsar un clave pro continuar..."

#: ../bin/apport-cli.py:94
msgid "What would you like to do? Your options are:"
msgstr "Quod amarea tu facer? Tu optiones es:"

#: ../bin/apport-cli.py:107
#, python-format
msgid "Please choose (%s):"
msgstr "Per favor optar (%s):"

#: ../bin/apport-cli.py:167
#, python-format
msgid "(%i bytes)"
msgstr "(%i bytes)"

#: ../bin/apport-cli.py:169 ../gtk/apport-gtk.py:136 ../kde/apport-kde.py:414
msgid "(binary data)"
msgstr "(datos binari)"

#: ../bin/apport-cli.py:209 ../gtk/apport-gtk.ui.h:6 ../kde/apport-kde.py:205
msgid "Send problem report to the developers?"
msgstr "Mandar le reporto de problema al disveloppatores?"

#: ../bin/apport-cli.py:211
msgid ""
"After the problem report has been sent, please fill out the form in the\n"
"automatically opened web browser."
msgstr ""
"Postquam le reporto de problema ha essite expedite, per favor impler le "
"modulo in le\n"
"navigator navigator aperite automaticamente."

#: ../bin/apport-cli.py:218
#, python-format
msgid "&Send report (%s)"
msgstr "&S Mandar le reporto (%s)"

#: ../bin/apport-cli.py:222
msgid "&Examine locally"
msgstr "&Examinar localmente"

#: ../bin/apport-cli.py:226
msgid "&View report"
msgstr "&Vider le reporto"

#: ../bin/apport-cli.py:228
msgid "&Keep report file for sending later or copying to somewhere else"
msgstr ""
"&K Tener le file de reporto pro le expedition subsequente o sinon copiar a "
"alicubi"

#: ../bin/apport-cli.py:231
msgid "Cancel and &ignore future crashes of this program version"
msgstr "Cancellar e &ignorar futur crashes de iste version de programma"

#: ../bin/apport-cli.py:234 ../bin/apport-cli.py:314 ../bin/apport-cli.py:350
#: ../bin/apport-cli.py:371
msgid "&Cancel"
msgstr "&Cancellar"

#: ../bin/apport-cli.py:255
msgid "Problem report file:"
msgstr "File reporto de problema:"

#: ../bin/apport-cli.py:261 ../bin/apport-cli.py:266
msgid "&Confirm"
msgstr "&Confirmar"

#: ../bin/apport-cli.py:265
#, python-format
msgid "Error: %s"
msgstr "Error: %s"

#: ../bin/apport-cli.py:271 ../kde/apport-kde.py:462
msgid "Collecting problem information"
msgstr "Collection del informationes del problema"

#: ../bin/apport-cli.py:273
msgid ""
"The collected information can be sent to the developers to improve the\n"
"application. This might take a few minutes."
msgstr ""
"Le informationes colligite pote esser expedite al disveloppatores pro "
"affinar le application. Isto pote prender alicun minutas."

#: ../bin/apport-cli.py:289 ../gtk/apport-gtk.ui.h:17 ../kde/apport-kde.py:513
msgid "Uploading problem information"
msgstr "Cargamento del informationes del problema"

#: ../bin/apport-cli.py:291
msgid ""
"The collected information is being sent to the bug tracking system.\n"
"This might take a few minutes."
msgstr ""
"Le informationes colligite pote esser expedite al systema de revision del "
"defectos. Isto pote prender alicun minutas."

#: ../bin/apport-cli.py:312
msgid "&Yes"
msgstr "Si &Y"

#: ../bin/apport-cli.py:313
msgid "&No"
msgstr "&No"

#: ../bin/apport-cli.py:349
msgid "&Done"
msgstr "&D Facite"

#: ../bin/apport-cli.py:355
msgid "none"
msgstr "necun"

#: ../bin/apport-cli.py:356
#, python-format
msgid "Selected: %s. Multiple choices:"
msgstr "Seligite: %s. Selectiones multiple:"

#: ../bin/apport-cli.py:372
msgid "Choices:"
msgstr "Selectiones:"

#: ../bin/apport-cli.py:386
msgid "Path to file (Enter to cancel):"
msgstr "Percurso al file (Entrar pro cancellar):"

#: ../bin/apport-cli.py:392
msgid "File does not exist."
msgstr "Le file non existe."

#: ../bin/apport-cli.py:394
msgid "This is a directory."
msgstr "Isto es un directorio."

#: ../bin/apport-cli.py:399
msgid "To continue, you must visit the following URL:"
msgstr "Pro continuar, tu debe visitar le adresse URL sequente:"

#: ../bin/apport-cli.py:401
msgid ""
"You can launch a browser now, or copy this URL into a browser on another "
"computer."
msgstr ""
"Tu pote lancear ora un navigator, o copiar iste adresse URL in un navigator "
"sur un altere computer."

#: ../bin/apport-cli.py:406
msgid "Launch a browser now"
msgstr "Lancear un navigator ora"

#: ../bin/apport-cli.py:421
msgid "No pending crash reports. Try --help for more information."
msgstr ""
"Nulle reporto de crash pendente. Probar --help pro plus de informationes."

#: ../bin/apport-retrace.py:48
msgid "Do not put the new traces into the report, but write them to stdout."
msgstr "Non poner le nove tracias in le reporto, ma scriber los al stdout."

#: ../bin/apport-retrace.py:55
msgid ""
"Start an interactive gdb session with the report's core dump (-o ignored; "
"does not rewrite report)"
msgstr ""
"Initiar un session gdb interactive con le core dump del reporto (-o "
"ignorate; non rescriber le reporto)"

#: ../bin/apport-retrace.py:64
msgid ""
"Write modified report to given file instead of changing the original report"
msgstr ""
"Scriber on reporto modificate pro le file date in vice de cambiar le reporto "
"original"

#: ../bin/apport-retrace.py:73
msgid "Remove the core dump from the report after stack trace regeneration"
msgstr ""
"Remover le core dump ex le reporto post le regeneration del tracias del pila"

#: ../bin/apport-retrace.py:76
msgid "Override report's CoreFile"
msgstr "Superscriber le CoreFile del reporto"

#: ../bin/apport-retrace.py:79
msgid "Override report's ExecutablePath"
msgstr "Superscriber le ExecutablePath del reporto"

#: ../bin/apport-retrace.py:82
msgid "Override report's ProcMaps"
msgstr "Superscriber le ProcMaps del reporto"

#: ../bin/apport-retrace.py:88
msgid "Rebuild report's Package information"
msgstr "Reproducer le informationes del pacchetto del reporto"

#: ../bin/apport-retrace.py:95
msgid ""
"Build a temporary sandbox and download/install the necessary packages and "
"debug symbols in there; without this option it assumes that the necessary "
"packages and debug symbols are already installed in the system. The argument "
"points to the packaging system configuration base directory; if you specify "
"\"system\", it will use the system configuration files, but will then only "
"be able to retrace crashes that happened on the currently running release."
msgstr ""
"Construer un arenario temporanee e  discargar/installar le pacchettos "
"necessari e le symbolos de correction ibi; sin iste option on assume que le "
"pacchettos necessari e le symbolos de correction es ja installate in le "
"systema. Le argumento puncta al todirectorio base del configuration del "
"systema de packaging; Si on specifica \"systema\", illo usara le files de "
"configuration del systema, ma alora essera habile sol a retraciar le crashes "
"que ha occurrite sur le edition fluente actualmente."

#: ../bin/apport-retrace.py:109
msgid ""
"Build another temporary sandbox for installing gdb and its dependencies "
"using the same release as the report rather than whatever version of gdb you "
"have installed."
msgstr ""
"Crear un altere alveo de arena temporari pro installar gdb e su dependentias "
"per le mesme edition del reporto, plu tosto que ulle version de gdb que tu "
"ha ja installate."

#: ../bin/apport-retrace.py:119
msgid "Report download/install progress when installing packages into sandbox"
msgstr ""
"Referer le progresso del discargamento/installation quando on installa un "
"pacchetto in le arenario"

#: ../bin/apport-retrace.py:125
msgid "Prepend timestamps to log messages, for batch operation"
msgstr "Prepend timestamps pro messages de registro, pro le operation de fundo"

#: ../bin/apport-retrace.py:131
msgid ""
"Create and use third-party repositories from origins specified in reports"
msgstr ""
"Crear e usar repositorios de tertie partes ex origines specificate in le "
"reportos"

#: ../bin/apport-retrace.py:139
msgid "Cache directory for packages downloaded in the sandbox"
msgstr "Directorio tampon pro le pacchettos discargate in le arenario"

#: ../bin/apport-retrace.py:145
msgid ""
"Directory for unpacked packages. Future runs will assume that any already "
"downloaded package is also extracted to this sandbox."
msgstr ""
"Directorio pro le expansion del pacchettos. In le futur functionamentos on "
"assumera que omne le pacchettos ja discargate essera extrahite in iste "
"arenario."

#: ../bin/apport-retrace.py:156 ../bin/apport-valgrind.py:99
msgid ""
"Install an extra package into the sandbox (can be specified multiple times)"
msgstr ""
"Installar un pacchetto extra in le arenario (pote esser specificate multiple "
"vices)"

#: ../bin/apport-retrace.py:163
msgid ""
"Path to a file with the crash database authentication information. This is "
"used when specifying a crash ID to upload the retraced stack traces (only if "
"neither -g, -o, nor -s are specified)"
msgstr ""
"Percurso a un file con le informationes de authentication del base de datos "
"del crash. Isto es usate pro specificar un ID de crash pro cargar le tracias "
"del pila retraciate (sol si ni -g, -o, ni -s es specificate)"

#: ../bin/apport-retrace.py:173
msgid ""
"Display retraced stack traces and ask for confirmation before sending them "
"to the crash database."
msgstr ""
"Monstrar le tracias del pila retraciate e quere pro le confirmation ante que "
"inviar los al base de datos del crash ."

#: ../bin/apport-retrace.py:181
msgid "Path to the duplicate sqlite database (default: no duplicate checking)"
msgstr ""
"Percurso al base de datos sqlite duplicate (standard: nulle controlo del "
"duplicatos)"

#: ../bin/apport-retrace.py:188
msgid "Do not add StacktraceSource to the report."
msgstr "Non adder StacktraceSource al reporto."

#: ../bin/apport-retrace.py:201
msgid "You cannot use -C without -S. Stopping."
msgstr "On non pote usar -C sin -S. Arrestar."

#. translators: don't translate y/n,
#. apport currently only checks for "y"
#: ../bin/apport-retrace.py:231
msgid "OK to send these as attachments? [y/n]"
msgstr "OK a inviar istes como attaccamentos? [s/n]"

#: ../bin/apport-unpack.py:35
#, python-format
msgid "%(prog)s <report> <target directory>"
msgstr "%(prog)s <report> <directorio de destination>"

#: ../bin/apport-unpack.py:36
msgid "Report file to unpack"
msgstr "File de reporto a dispacchettar"

#: ../bin/apport-unpack.py:37
msgid "directory to unpack report to"
msgstr "directorio  pro dispacchettar le reporto"

#: ../bin/apport-unpack.py:86
msgid "Destination directory exists and is not empty."
msgstr "Le directorio de destination existe e non es vacue."

#: ../bin/apport-valgrind.py:40
msgid "See man page for details."
msgstr "Vider pro le detalios le pagina principal."

#: ../bin/apport-valgrind.py:49
msgid "specify the log file name produced by valgrind"
msgstr "attribuer le nomine del file registro producite per valgrind"

#: ../bin/apport-valgrind.py:55
msgid ""
"reuse a previously created sandbox dir (SDIR) or, if it does not exist, "
"create it"
msgstr ""
"reusar un directorio (SDIR) del arenario create anteriormente o, si illo non "
"existe, crear lo"

#: ../bin/apport-valgrind.py:63
msgid ""
"do  not  create  or reuse a sandbox directory for additional debug symbols "
"but rely only on installed debug symbols."
msgstr ""
"non crear o reusar un directorio del arenario o symbolos additional de "
"correctiones sed fider sol sur le symbolos de correction ja installate."

#: ../bin/apport-valgrind.py:72
msgid ""
"reuse a previously created cache dir (CDIR) or, if it does not exist, create "
"it"
msgstr ""
"directorio (SDIR) del memoria tampon create anteriormente o, si illo non "
"existe, crear lo"

#: ../bin/apport-valgrind.py:81
msgid "report download/install progress when installing packages into sandbox"
msgstr ""
"referer le progresso del discargamento/installation quando on installa un "
"pacchetto in le arenario"

#: ../bin/apport-valgrind.py:88
msgid ""
"the executable that is run under valgrind's memcheck tool for memory leak "
"detection"
msgstr ""
"le executabile que es exequite sub le application memcheck de valgrind pro "
"le detection del perdita de memoria"

#: ../bin/apport-valgrind.py:130
#, python-format
msgid "Error: %s is not an executable. Stopping."
msgstr "Error: %s non es un  executabile. Arresto."

#: ../data/apportcheckresume.py:75
msgid ""
"This occurred during a previous suspend, and prevented the system from "
"resuming properly."
msgstr ""
"Iato occurreva durante un previe suspension e ha prevenite le proprie "
"reinitio del systema."

#: ../data/apportcheckresume.py:80
msgid ""
"This occurred during a previous hibernation, and prevented the system from "
"resuming properly."
msgstr ""
"Iato occurreva durante un previe hibernation e ha prevenite le proprie "
"reinitio del systema."

#: ../data/apportcheckresume.py:88
msgid ""
"The resume processing hung very near the end and will have appeared to have "
"completed normally."
msgstr ""
"Le processo de resumer hung multo proxime al fin e essera apparite haber "
"completate normalmente."

#: ../data/kernel_oops.py:33
msgid "Your system might become unstable now and might need to be restarted."
msgstr ""
"Tu systema pote devenir instabile ora e pote necessitar ser reinitiate."

#: ../gtk/apport-gtk.desktop.in.h:1 ../kde/apport-kde-mime.desktop.in.h:1
#: ../kde/apport-kde.desktop.in.h:1
msgid "Report a problem..."
msgstr "Reportar un problema..."

#: ../gtk/apport-gtk.desktop.in.h:2 ../kde/apport-kde-mime.desktop.in.h:2
#: ../kde/apport-kde.desktop.in.h:2
msgid "Report a malfunction to the developers"
msgstr "Reportar un dysfunction al disveloppatores"

#: ../gtk/apport-gtk.py:153
#, python-format
msgid "Sorry, the application %s has stopped unexpectedly."
msgstr "Displacente, le application %s ha stoppate inexpectatemente."

#: ../gtk/apport-gtk.py:156
#, python-format
msgid "Sorry, %s has closed unexpectedly."
msgstr "Displacente, %s ha claudite inexpectatemente."

#: ../gtk/apport-gtk.py:161 ../kde/apport-kde.py:220 ../kde/apport-kde.py:269
#, python-format
msgid "Sorry, %s has experienced an internal error."
msgstr "Displacente, %s ha experimentate un error interne."

#: ../gtk/apport-gtk.py:184 ../gtk/apport-gtk.ui.h:14 ../kde/apport-kde.py:213
msgid "Send"
msgstr "Mandar"

#: ../gtk/apport-gtk.py:205 ../gtk/apport-gtk.py:619 ../gtk/apport-gtk.ui.h:11
#: ../kde/apport-kde.py:330
msgid "Show Details"
msgstr "Monstrar le detalios"

#: ../gtk/apport-gtk.py:218 ../kde/apport-kde.py:261 ../kde/apport-kde.py:276
msgid "Continue"
msgstr "Continuar"

#: ../gtk/apport-gtk.py:242
#, python-format
msgid "The application %s has stopped responding."
msgstr "Le application %s ha stoppate de responder."

#: ../gtk/apport-gtk.py:246
#, python-format
msgid "The program \"%s\" has stopped responding."
msgstr "Le programma %s ha stoppate de responder."

#: ../gtk/apport-gtk.py:263 ../kde/apport-kde.py:228
#, python-format
msgid "Package: %s"
msgstr "Pacchetto: %s"

#: ../gtk/apport-gtk.py:269 ../kde/apport-kde.py:235
msgid "Sorry, a problem occurred while installing software."
msgstr ""
"Displacente, un problema occurreva durante le installation del software."

#: ../gtk/apport-gtk.py:279 ../gtk/apport-gtk.py:298 ../kde/apport-kde.py:243
#, python-format
msgid "The application %s has experienced an internal error."
msgstr "Le application %s ha experimentate un  error interne."

#: ../gtk/apport-gtk.py:281 ../kde/apport-kde.py:248
#, python-format
msgid "The application %s has closed unexpectedly."
msgstr "Le application %s ha claudite inexpectatemente."

#: ../gtk/apport-gtk.py:306 ../gtk/apport-gtk.ui.h:7 ../kde/apport-kde.py:273
msgid "If you notice further problems, try restarting the computer."
msgstr "Si on nota problemas additional, provar a reinitiar le computer."

#: ../gtk/apport-gtk.py:309 ../kde/apport-kde.py:278
msgid "Ignore future problems of this type"
msgstr "Ignorar le problema futur de iste typo"

#: ../gtk/apport-gtk.py:623 ../kde/apport-kde.py:327
msgid "Hide Details"
msgstr "Occultar le detalios"

#: ../gtk/apport-gtk.ui.h:1 ../kde/apport-kde.py:381 ../kde/apport-kde.py:555
msgid "Apport"
msgstr "Apport"

#: ../gtk/apport-gtk.ui.h:2
msgid "Cancel"
msgstr "Cancellar"

#: ../gtk/apport-gtk.ui.h:3
msgid "OK"
msgstr "OK"

#: ../gtk/apport-gtk.ui.h:4
msgid "Crash report"
msgstr "Reporto de disastro"

#: ../gtk/apport-gtk.ui.h:5
msgid "<big><b>Sorry, an internal error happened.</b></big>"
msgstr "<big><b>Displacente, ha occurrite un error interne.</b></big>"

#: ../gtk/apport-gtk.ui.h:8
msgid "Remember this in future"
msgstr "Rememorar isto in futuro"

#: ../gtk/apport-gtk.ui.h:9
msgid "Ignore future problems of this program version"
msgstr "Ignorar le futur problema de iste version del programma"

#: ../gtk/apport-gtk.ui.h:10
msgid "Relaunch this application"
msgstr "Relancear iste application"

#: ../gtk/apport-gtk.ui.h:12
msgid "_Examine locally"
msgstr "_Examinar localmente"

#: ../gtk/apport-gtk.ui.h:13
msgid "Don't send"
msgstr "non inviar"

#: ../gtk/apport-gtk.ui.h:15
msgid "<big><b>Collecting problem information</b></big>"
msgstr "<big><b>Collection del informationes del problema</b></big>"

#: ../gtk/apport-gtk.ui.h:16
msgid ""
"Information is being collected that may help the developers fix the problem "
"you report."
msgstr ""
"On va colliger informationes que pote adjutar le disveloppatores a fixar le "
"problema que on reporta."

#: ../gtk/apport-gtk.ui.h:18
msgid "<big><b>Uploading problem information</b></big>"
msgstr "<big><b>Cargamento del informationes del problema</b></big>"

#: ../gtk/apport-gtk.ui.h:19 ../kde/apport-kde.py:515
msgid ""
"The collected information is being sent to the bug tracking system. This "
"might take a few minutes."
msgstr ""
"Le informationes colligite pote esser expedite al systema de revision del "
"defectos. Isto pote prender alicun minutas."

#: ../kde/apport-kde-mimelnk.desktop.in.h:1
msgid "Apport crash file"
msgstr "File crash de Apport"

#: ../kde/apport-kde.py:264
msgid "Leave Closed"
msgstr "Lassar clause"

#: ../kde/apport-kde.py:265 ../kde/apport-kde.py:432
msgid "Relaunch"
msgstr "Relancear"

#: ../kde/apport-kde.py:358
msgid "Username:"
msgstr "Nomine del usator:"

#: ../kde/apport-kde.py:361
msgid "Password:"
msgstr "Contrasigno:"

#: ../kde/apport-kde.py:461
msgid "Collecting Problem Information"
msgstr "Collection del informationes del problema"

#: ../kde/apport-kde.py:464
msgid ""
"The collected information can be sent to the developers to improve the "
"application. This might take a few minutes."
msgstr ""
"Le informationes colligite pote esser expedite al disveloppatores pro "
"affinar le application. Isto pote prender alicun minutas."

#: ../kde/apport-kde.py:512
msgid "Uploading Problem Information"
msgstr "Cargamento del informationes del problema"
